### ISSUE TITLE
Gremlin query - Apache TinkerPop - HelloWorld execution example

### ISSUE THREAD

**Issue by authorjapps** [authorjapps/zerocode-hello-world#5]
@mailtoach79 to put an example i.e. a Step executing gremlin query.
Either via a `string` body or via `$PATH:<path_to_file>`

This is already supported by the framework. 👍 

But if a specialized `place-holder` needed like below-
e.g. `$GREMLIN.QUERY.FILE:<path to file>`, then it has to be created.

Mr Uma has to suggest !

**Comment by mailtoach79** [comment]
Yes, we can pass gremlin in two ways as explained above. I would update the hello world execution example soon. thanks.
